_includes/default-footer.html

```diff
@@ -44,6 +44,9 @@ <h3>Get in touch</h3>
             <a href="https://github.com/htmluke">Lucas Emanoell Medeiros e Silva</a> and
             then adapted by the team.
           </p>
+          <p>
+            Open Life Science Limited is a non-profit charitable company limited by guarantee, based in the United Kingdom. Company number: 12824090
```

Comment:
Thank you @emmyft <3. The CI fails because of ":" (a super minor error that I have made before), updating that with another symbol might fix it.